src/Symfony/Bundle/FrameworkBundle/Command/CachePoolDeleteCommand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\HttpKernel\CacheClearer\Psr6CacheClearer;
+
+/**
+ * Delete an item from a cache pool.
+ *
+ * @author Pierre du Plessis <pdples@gmail.com>
+ */
+final class CachePoolDeleteCommand extends Command
+{
+    protected static $defaultName = 'cache:pool:delete';
+
+    private $poolClearer;
+
+    public function __construct(Psr6CacheClearer $poolClearer)
```

Comment:
Do we really want to inject the cache clearer to then use it to fetch a pool? Shouldn't we rather inject the pools directly?